snuba/admin/static/api_client.tsx

```diff
@@ -95,14 +95,17 @@ function Client() {
       );
     },
     getClickhouseCannedQueries: () => {
-      const url = baseUrl + "clickhouse_queries";
+      const url = baseUrl + 'clickhouse_queries';
       return fetch(url).then((resp) => resp.json());
     },
-    executeQuery: (query: QueryRequest) => {
-      const url = baseUrl + "run_clickhouse_system_query";
+    executeQuery: (query: QueryRequest, endpoint?: string) => {
+      const endpoint_to_use = endpoint
+        ? endpoint
+        : 'run_clickhouse_system_query';
```

Comment:
It's not super important but I'd almost prefer this to require the argument (and make the change in the other file)

Reply:
also for this pattern you can shorten it to `endpoint ?? 'run_clickhouse_system_query'`